src/components/CubeListNavbar.js

```diff
@@ -440,13 +440,21 @@ const CubeListNavbar = ({
                   </>
                 )}
                 <DropdownItem href={`/cube/clone/${cubeID}`}>Clone Cube</DropdownItem>
-                <DropdownItem href={`/cube/download/plaintext/${cubeID}`}>Card Names (.txt)</DropdownItem>
+                <DropdownItem href={`/cube/download/plaintext/${cubeID}?${sortUrlSegment}${filterUrlSegment}`}>
```

Comment:
This is minor, but since we're now using it everywhere, it might be neater to define the whole parameter string into a variable so that the same three segments don't have to be repeated.